third_party/forked/reflect/deep_equal.go

```diff
@@ -156,7 +156,10 @@ func (e Equalities) deepValueEqual(v1, v2 reflect.Value, visited map[visit]bool,
 		}
 		return true
 	case reflect.Slice:
-		if (v1.IsNil() || v1.Len() == 0) != (v2.IsNil() || v2.Len() == 0) {
+		// go1.5: simplified if to work around https://github.com/golang/go/issues/12266
```

Comment:
Let's not have these workarounds in our code, please--let's wait for a go1.5 patch before using go1.5